crates/bws/src/main.rs

```diff
@@ -320,12 +323,20 @@ async fn process_commands() -> Result<()> {
         })
         .transpose()?;
 
+    let state_file_path = state::get_state_file_path(
+        profile.and_then(|p| p.state_file_dir.map(|sfd| PathBuf::from(sfd.clone()))),
```

Comment:
```suggestion
        profile.and_then(|p| p.state_file_dir).map(Into::into),
```

Reply:
Resolved in [16703f4](https://github.com/bitwarden/sdk/pull/388/commits/16703f4eebf7cc48a1de24840e5ede3832864590)